client/state/types.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * External dependencies
+ */
+import { DefaultRootState } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { IPluginsState } from 'calypso/state/plugins/reducer';
+import { IMarketplaceState } from 'calypso/state/marketplace/types';
+
+export interface IAppState extends DefaultRootState {
```

Comment:
Would be nice if we add a comment that this type is incomplete and is a work in progress.